base/src/y2base.cc

```diff
@@ -1,22 +0,0 @@
-/*---------------------------------------------------------------------\
-|                                                                      |
-|                      __   __    ____ _____ ____                      |
-|                      \ \ / /_ _/ ___|_   _|___ \                     |
-|                       \ V / _` \___ \ | |   __) |                    |
-|                        | | (_| |___) || |  / __/                     |
-|                        |_|\__,_|____/ |_| |_____|                    |
-|                                                                      |
-|                               core system                            |
-|                                                        (C) SuSE GmbH |
-\----------------------------------------------------------------------/
-
-   File:       y2base.cc
-
-   Author:     Mathias Kettner <kettner@suse.de>
-   Maintainer: Mathias Kettner <kettner@suse.de>
-
-/-*/
-/*
- * Hi. This file is empty on purpose :-)
- * Nice, isn't it?
```

Comment:
Well, that means we are not really dropping the old code. The old code is in [liby2/src/genericfrontend.cc](https://github.com/yast/yast-core/blob/a0fefb49b7497a21624fd6f65a35de56b2dbe74a/liby2/src/genericfrontend.cc). Why don't we drop that too?